package-lock.json

```diff
@@ -17,12 +17,13 @@
                 "@azure/arm-resources": "^5.2.0",
                 "@azure/container-registry": "1.0.0-beta.5",
                 "@azure/core-rest-pipeline": "1.10.3",
+                "@azure/monitor-query": "^1.2.0",
                 "@azure/storage-blob": "^12.4.1",
                 "@fluentui/react-components": "^9.56.2",
                 "@fluentui/react-icons": "^2.0.265",
                 "@microsoft/vscode-azext-azureutils": "^3.3.3",
                 "@microsoft/vscode-azext-github": "^1.0.2",
-                "@microsoft/vscode-azext-utils": "^3.2.1",
+                "@microsoft/vscode-azext-utils": "^3.3.0",
```

Comment:
Whoa.. any idea why there are 7273 lines of code deleted? Kind of wild.

Reply:
It's from Azure Monitor Query, it blew my mind too, I don't really understand how it led to so many lines removed lol

Reply:
Did it change the lockfile version?